curations/maven/mavencentral/jakarta.ejb/jakarta.ejb-api.yaml

```diff
@@ -0,0 +1,24 @@
+coordinates:
+  name: jakarta.ejb-api
+  namespace: jakarta.ejb
+  provider: mavencentral
+  type: maven
+revisions:
+  3.2.5:
+    described:
+      sourceLocation:
+        name: ejb-api
+        namespace: eclipse-ee4j
+        provider: github
+        revision: 5d6d38591e977d876b1d57865d75142374a1859c
+        type: git
+        url: 'https://github.com/eclipse-ee4j/ejb-api/commit/5d6d38591e977d876b1d57865d75142374a1859c'
+    files:
+      - license: ' EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 '
```

Comment:
There is something really weird with the leading space in this string. @storrisi @ignacionr would you know if this could have been a user mistake or a bug?

Reply:
Several unit tests aim at the merging of licenses, so it'd be very strange that this would have come from automatic merge. It seems manual to me.